backend/lib/edgehog/devices/device/types/led_behavior.ex

```diff
@@ -0,0 +1,30 @@
+#
+# This file is part of Edgehog.
+#
+# Copyright 2024 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+defmodule Edgehog.Devices.Device.Types.LedBehavior do
```

Comment:
Riccardo initially defined a `Types` folder for Astarte values of devices, however we eventually avoided this approach for further developments.
It's probably not necessary to indicate that this file contains a *type*, distinguishing it from other fields that might be implemented as embedded resources or in other ways.

We could just keep `led_behavior.ex` next to `device.ex` for now.
 
```suggestion
defmodule Edgehog.Devices.Device.LedBehavior do
```